cinema/views.py

```diff
@@ -57,7 +62,7 @@ class CinemaHallViewSet(
 ):
     queryset = CinemaHall.objects.all()
     serializer_class = CinemaHallSerializer
-    authentication_classes = (TokenAuthentication,)
+    authentication_classes = (JWTAuthentication,)
```

Comment:
can you just add in settings `DEFAULT_AUTH_CLASSES` so you don't need to write it in every view ?